GiftGivingGenerator.API/Repositories/GiftWishRepository.cs

```diff
@@ -18,4 +17,13 @@ public GiftWish GetByEventAndPerson(Guid eventId, Guid personsId)
 		
 		return giftWish;
 	}
+	public void RemoveAll(Guid eventId, Guid personId)
```

Comment:
why the name is RemoveAll but you're looking for a single wish?

Reply:
Because I've changed conception. Now I corrected name.